src/state/store/initialState.js

```diff
@@ -1,6 +1,13 @@
 const initialState = {
   lat: 0,
   lng: 0,
+  name: null,
+  authenticated: false,
+  displayLoginButton: true,
+  displaySignupButton: true,
+  authMessage: null,
+  userAttrs: null,
+  message: null,
```

Comment:
I don't think we need to store the user attributes and message already exists and you're not using it

Reply:
done